lib/reckon/money.rb

```diff
@@ -43,11 +43,19 @@ def pretty( negate = false )
     end
 
     def Money::from_s( value, options = {} )
-      return nil if value.empty?
+      return Money.new( 0.00, options ) if value.empty?
       value = value.gsub(/\./, '').gsub(/,/, '.') if options[:comma_separates_cents]
-      amount = value.gsub(/[^\d\.]/, '').to_f
-      amount *= -1 if value =~ /[\(\-]/
-      Money.new( amount, options )
+      value = value.gsub(/,/, '')
+      m = value.match( /(\D*)(\d+\.\d\d)(\D*)/ ) || value.match(/^(.*?)([\d\.]+)(\D*)$/)
+      if m 
+        amount = m[2].to_f
```

Comment:
A comment about what this method is doing might be helpful.  I'm having trouble following it.